e2e/libs/keywords/common_keywords.py

```diff
@@ -1,12 +1,20 @@
-import logging
 from utility.utility import init_k8s_api_client
-import utils.common_utils
+from node_exec import NodeExec
+import logging
 
 class common_keywords:
 
-    def set_test_environment(self):
+    def __init__(self):
+        logging.warn("initialize common_keywords class")
+
+    def init_k8s_api_client(self):
         init_k8s_api_client()
 
-    def generate_support_bundle(self, case_name):
-        logging.info(f"generating support bundle for {case_name}")
-        common_utils.generate_support_bundle(case_name)
+    def init_node_exec(self, test_name):
+        namespace = test_name.lower().replace(' ', '-')[:63]
+        logging.warn(f"namespace = {namespace}")
```

Comment:
As mentioned before, need to review log level later. Just need to follow log level to print messages with right level.